projects/Mallard/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
```

Comment:
@groakland had a look at the build failure, it looks like this gradle update is bringing new changes which are not compatible with gradle setup for ophan android project. I'm not sure it would be a trivial fix so wondering how bad it would be if we revert this ReactNative upgrade? I don't like the idea of reverting back react-native but it would be good to know where this this RN upgrade was done to fix some other issues or it was just to keep things updated?